Assets/LeapMotionModules/Interaction/InteractionHand.cs

```diff
@@ -218,8 +229,7 @@ private struct HoverCheckResults {
 
     private void FixedUpdateTouch() {
       _touchActivityManager.activationRadius = interactionManager.WorldTouchActivationRadius;
-
-      _touchActivityManager.FixedUpdateHand(_hand);
+      _touchActivityManager.FixedUpdateHand((_hand != null) ? _hand.PalmPosition.ToVector3() : Vector3.zero);
```

Comment:
Wait if the hand is null, do we want to update activity at all?  I would think updating the activity assuming the hand is at the origin might result in some strange behavior